subsys/bluetooth/mesh/light_ctrl_srv.c

```diff
@@ -470,7 +470,7 @@ static void prolong(struct bt_mesh_light_ctrl_srv *srv)
 		return;
 	}
 
-	BT_DBG("");
+	BT_DBG("Light Prolonged");
```

Comment:
These shouldn't be necessary if `CONFIG_LOG_MINIMAL=n`, as the function name will be printed

Reply:
The case was that I just added debug, and then I got a lot of "D:", and that looks very strange. I believe that it something that the user also will do. I did not change set CONFIG_LOG_MINIMAL=n in the beginning, so by adding this it helped me in the beginning. But I can remove all these changes in this file, as they have nothing to do with this task.

Reply:
The problem was that we could not use the CONFIG_LOG_MINIMAL when running unit test. So to get some sensible debug logs when running unit test, I needed to add these texts. I you insist I can remove them.